src/components/AdvanceAnalyticsV2/AnalyticsV2Page.jsx

```diff
@@ -35,10 +36,7 @@ const AnalyticsV2Page = ({ enterpriseId }) => {
   });
   const showWarningBanner = cookies.get(ANALYTICS_WARNING_BANNER_COOKIE);
   const currentDate = new Date().toISOString().split('T')[0];
-  const formatDate = (dateString) => {
-    const options = { year: 'numeric', month: 'long', day: 'numeric' };
-    return new Date(dateString).toLocaleDateString(undefined, options);
-  };
+  const defaultDataUpdatedDate = new Date().toISOString();
```

Comment:
[question/clarification] Can you help me understand the interaction between the existing `currentDate` and the added `defaultDataUpdatedDate`? Is it intended to continue using `currentDate` variable in various places throughout this component? Should we be using only one or the other, since they are largely both representing the same thing (the current timestamp)?